src/window/building/distribution.c

```diff
@@ -251,6 +348,19 @@ int window_building_handle_mouse_dock_orders(const mouse* m, building_info_conte
     return generic_buttons_handle_mouse(m, c->x_offset + 80, y_offset + 404, market_order_buttons, 1, &data.orders_focus_button_id);
 }
 
+static void on_dock_cities_scroll() {
+  dock_cities_set_scroll_position(dock_scrollbar.scroll_position);
+  window_invalidate();
+}
+
+static int dock_cities_scroll_position() {
+  return data.dock_scrollbar_position;
+}
+
+void dock_cities_set_scroll_position(int scroll_position) {
+  data.dock_scrollbar_position = scroll_position;
+}
```

Comment:
Styling issues on the previous three lines:

- Identation is wrong
- The braces after a function definition are on the next line

Reply:
@crudelios then the rule is to never have the opening brace just after function definition, and always add newline before brace? is it only for functions or also for any block?

Reply:
Here's a small example with some of the ideal formatting applied:

```
int string_length(const char *string)
{
    if (!string) {
        return 0;
    }
    int length = 0;
    while (*string++) {
        length++;
    }
    return length;
}
```

Reply:
This is also resolved, I'm not marking it as such so you can check my previous comment.

Reply:
Got it, let’s go